webgui/InstallGUI.cmake

```diff
@@ -2,7 +2,31 @@ message("Install npm packages...")
 
 execute_process(
   COMMAND cp ${CC_PACKAGE} ${INSTALL_SCRIPTS_DIR}/package.json
+  WORKING_DIRECTORY ${INSTALL_SCRIPTS_DIR})
+
+execute_process(
   COMMAND npm install
   WORKING_DIRECTORY ${INSTALL_SCRIPTS_DIR})
 
-message("Installation of npm packages are finished.")
\ No newline at end of file
+execute_process(
+  COMMAND ${CMAKE_COMMAND} -E create_symlink dojo-util node_modules/util
+  WORKING_DIRECTORY ${INSTALL_SCRIPTS_DIR})
+
+message("Installation of npm packages are finished.")
+
+message("Initiating the Dojo Build System...")
+
+if (CMAKE_BUILD_TYPE STREQUAL "Release")
+  set(DOJO_OPTIMIZE "")
+  set(DOJO_LAYEROPTIMIZE "closure")
+else()
+  set(DOJO_OPTIMIZE "")
+  set(DOJO_LAYEROPTIMIZE "")
+endif()
```

Comment:
That is the optimization parameter for the non-layered files. Setting it to `closure` will add an extra ~60 sec to installation time, while the runtime performance will not be improved noticeably, since the JS resources are well bundled (layered) in our case. So I left it disabled also in the release mode.

Reply:
We can completly remove the `DOJO_OPTIMIZE` variable, if it is confusing.

Reply:
CMake `set()` calls can have a `DESCRIPTION`... maybe it's worth to leave this variable in (but define *outside* the `if` to empty-string) and put a description to both `_OPTIMIZE` and `_LAYEROPTIMIZE`?

Reply:
Modified accordingly. (I have not found a `DESCRIPTION` option without making the variable cached, but have added a comment to the code.)